src/Appwrite/Auth/OAuth2/Linkedin.php

```diff
@@ -144,9 +144,8 @@ public function getUserEmail(string $accessToken): string
      */
     public function isEmailVerified(string $accessToken): bool
     {
-        $email = $this->getUserEmail($accessToken);
-
-        return !empty($email);
+        $user = $this->getUser($accessToken);
+        return $user['email_verified'] ?? '';
```

Comment:
the function signature says this should return bool, but this line hear means it's possible for it to return a string.

```suggestion
        return $user['email_verified'] ?? false;
```

Reply:
Theoretically, an exception would be thrown elsewhere. For now, to keep things consistent, let's just return false.